_includes/main_nav.html

```diff
@@ -3,23 +3,29 @@
     <a class="navbar-brand" href="/"><img src="/images/bisq-logo.svg" height="30"/></a>
```

Comment:
When viewing the /ja/dao/ page, clicking on the Bisq logo should take me to /ja/ not /